pyquil/api/_engagement_manager.py

```diff
@@ -62,6 +75,9 @@ def get_engagement(
         If an engagement was already fetched previously and remains valid, it will be returned instead
         of creating a new engagement.
 
+        If an engagement cannot be created because the QPU is unavailable, a `QPUUnavailableError`
+        exception is raised, which includes the time to wait before retrying.
+
```

Comment:
I believe there is a `:raises ` annotation similar to `:return:` that will create a section for exceptions in the docs.